### PR TITLE
Cucumber-JVM 4.0.0 Announcement

### PR DESCRIPTION
**Pull request by mpkorstanje**

### CODE REVIEW

**Review by mattwynne — COMMENTED** [review]
\o/ well done everyone! 😀

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-4-0-0.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+Notable Changes
+----------------
+
+## --parallel ##
+
+Cucumber-JVM now supports parallel execution of Pickles[1] across threads! See the 
+documentation for each runner on how to enable this[2][3][4]. This was made possible 
+by some serious help from Kiel Boatman. Thanks!
+
+While executing in parallel each thread will have it's own seperate test context.
```
Comment (koic) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-4-0-0.md]: `seperate` seems to be a typo in `separate`.

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-4-0-0.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+refactor to using dependency injection first.
+
+Because pickles are now executed concurrently `TestCaseEvent`s will no longer 
+arrive in order. To mitigate the impact of this for `Eventlistener` implementations
+the `Eventbus` will record all events and replay them in a canonical order. 
+Implementations of `ConcurrentEventListener` will receive events as they happen. To help 
+concurrent event listners handle concurrent events all `TestCaseEvents` now also include 
+the `TestCase` of which they're part.
+
+1. A Pickle is a compiled Gherkin Scenario. A Scenario with Examples results in multiple Pickles, one for each Scenario.
```
Comment (jakzal) [inline apps/dynamic/views/_posts/announcing-cucumber-jvm-4-0-0.md]: Shouldn't `one for each Scenario` be `one for each Example`?

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-4-0-0.md (hunk 1 above)
Comment (mpkorstanje) [reply apps/dynamic/views/_posts/announcing-cucumber-jvm-4-0-0.md]: Thanks!

File: apps/dynamic/views/_posts/announcing-cucumber-jvm-4-0-0.md (hunk 2 above)
Comment (mpkorstanje) [reply apps/dynamic/views/_posts/announcing-cucumber-jvm-4-0-0.md]: That is correct.

**Pull request merged**